src/app/machines/views/MachineDetails/MachineNetwork/NetworkNotifications/NetworkNotifications.test.tsx

```diff
@@ -54,17 +48,11 @@ describe("NetworkNotifications", () => {
         system_id: "abc123",
       }),
     ];
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/machine/abc123", key: "testKey" }]}
-        >
-          <NetworkNotifications id="abc123" />
-        </MemoryRouter>
-      </Provider>
-    );
-    expect(wrapper.find("Notification").exists()).toBe(false);
+    renderWithBrowserRouter(<NetworkNotifications id="abc123" />, {
+      route: "/machine/abc123",
+      state,
+    });
+    expect(screen.queryByRole("notification")).toBeNull();
```

Comment:
We generally use more semantic `.not.toBeInTheDocument` for asserting that an element does not exist.

```suggestion
    expect(screen.queryByRole("notification")).not.toBeInTheDocument();
```

Reply:
Agh, this must have been a leftover from GPT. It was *obsessed* with `.toBeNull()`